props/saber_fett263_buttons.h

```diff
@@ -734,6 +734,20 @@ SaberFett263Buttons() : PropBase() {}
         return true;
 
 #endif
+        
+#ifdef FETT263_FORCE_PUSH_ALWAYS_ON
```

Comment:
This is so similar to the previous if statement.
I think you should write it as:

#ifdef FETT263_FORCE_PUSH_ALWAYS_ON
#define FORCE_PUSH_CONDITION true
#define FETT263_FORCE_PUSH
#else
#define FORCE_PUSH_CONDITION battle_mode_
#endif

#ifdef FETT263_FORCE_PUSH
          case EVENTID(BUTTON_NONE, EVENT_PUSH, MODE_ON):
￼        if (FORCE_PUSH_CONDITION && millis() - last_push_ > 2000) {
￼          if (SFX_push) {
￼            hybrid_font.PlayCommon(&SFX_push);
￼          } else {
￼            hybrid_font.DoEffect(EFFECT_FORCE, 0);
￼          }
￼          last_push_ = millis();
￼        }
￼        return true;
￼#endif

Reply:
thanks, I have updated